libr/core/cmd_anal.c

```diff
@@ -8258,14 +8295,15 @@ R_API void r_core_agraph_print (RCore *core, int use_utf, const char *input) {
 	case 'd': { // "aggd" - dot format
 		const char *font = r_config_get (core->config, "graph.font");
 		r_cons_printf ("digraph code {\nrankdir=LR;\noutputorder=edgesfirst\ngraph [bgcolor=azure];\n"
-			"edge [arrowhead=normal, color=\"#3030c0\" style=bold weight=2];\n"
-			"node [fillcolor=white, style=filled shape=box "
-			"fontname=\"%s\" fontsize=\"8\"];\n", font);
+			       "edge [arrowhead=normal, color=\"#3030c0\" style=bold weight=2];\n"
```

Comment:
no need to change these lines, they were right as they were before AFAIK.

Reply:
Clang format decided to change that, reverted.